lib/Mojo/Redis.pm

```diff
@@ -32,6 +32,11 @@ has pubsub => sub {
 };
 
 has url => sub { Mojo::URL->new($ENV{MOJO_REDIS_URL}) };
+has 'tls';
+has 'tls_ca';
+has 'tls_cert';
+has 'tls_key';
+has 'tls_options';
```

Comment:
I would rather have this part of the "url", like https://docs.mojolicious.org/Mojo/Server/Daemon#listen

Example:

```
Mojo::URL->new("redis://localhost?ca=ca.pem&cert=cert.pemt&key=key.pm")
```

And "tls" can be set to true if key and cert exists.